Makefile

```diff
@@ -0,0 +1,157 @@
+# Makefile for shipping and testing the container image.
+
+MAKEFLAGS += --warn-undefined-variables
+.DEFAULT_GOAL := build
+.PHONY: *
+
+# we get these from CI environment if available, otherwise from git
+GIT_COMMIT ?= $(shell git rev-parse --short HEAD)
+GIT_BRANCH ?= $(shell git rev-parse --abbrev-ref HEAD)
+WORKSPACE ?= $(shell pwd)
+
+namespace ?= autopilotpattern
+tag := branch-$(shell basename $(GIT_BRANCH))
+image := $(namespace)/prometheus
+testImage := $(namespace)/prometheus-testrunner
+
+#dockerLocal := DOCKER_HOST= DOCKER_TLS_VERIFY= DOCKER_CERT_PATH= docker
```

Comment:
Hm, that's unfortunate and is because Docker Machine uses `DOCKER_HOST`, etc. and Docker for Mac does not. The reason we're doing this is because we want the person (or Jenkins job) running the tests to be able to set their credentials to point to the environment-under-test but then run the tests from the local machine so that the keys never leave the machine running the tests.

This will effect every blueprint where we're doing testing this way. I'm certainly open to new suggestions on how to tackle it though.